virt_lightning/virt_lightning.py

```diff
@@ -937,9 +937,6 @@ def attach_network(
         for i, iface in enumerate(ifaces):
             self.nics[i]["mac"] = iface.attrib["address"]
 
-    def add_root_disk(self, root_disk_path):
```

Comment:
The API is supposed to be stable. Can you please keep the method.

Reply:
There was only one usage which I removed. So, now there are no usages. Should I keep it?

Reply:
I use it here: https://github.com/virt-lightning/ansible-collection-virt-lightning/blob/master/plugins/modules/vm.py#L86

This is the module for Ansible.

Reply:
Done.